File: problems/0/0/2/002.go
// Copyright (C) 2014 Jorge Aparicio

package main

import "fmt"

func main() {
    ans := 0
    curr := 1
    next := 2

    for curr < 4000000 {
        if curr % 2 == 0 {
            ans += curr
        }

        tmp := next
        next += curr
        curr = tmp
    }

    fmt.Println(ans)
}

File: problems/0/0/2/002.js
// Copyright (C) 2014 Jorge Aparicio

var ans = 0;
var curr = 1;
var next = 2;

while (curr < 4000000) {
  if (curr % 2 == 0) {
    ans += curr;
  }

  var tmp = next;
  next += curr;
  curr = tmp;
}

console.log(ans);

